build.assets/Makefile

```diff
@@ -109,7 +109,7 @@ buildbox:
 .PHONY:buildbox-fips
 buildbox-fips:
```

Comment:
optional: is it possible to squash all these permutations of the buildbox with something like:
```
buildbox-fips: BUILDBOX="$(BUILDBOX_FIPS)"
buildbox-fips: BUILDBOX_NAME="$(BUILDBOX_FIPS_NAME)"
buildbox-fips: buildbox
```

Reply:
I tried to do this lots of different ways before I started doing it this way, but it doesn't seem possible without removing the `buildbox` target as a dependency of `release` (which would probably hamper people's ability to use this locally). Each buildbox has a different Dockerfile and slightly different build arguments too which makes it harder.

There's probably a much smarter way to do this that I'm missing.